t/t0095-bloom.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/sh
+
+test_description='test bloom.c'
+. ./test-lib.sh
+
+test_expect_success 'get bloom filters for commit with no changes' '
+	git init &&
+	git commit --allow-empty -m "c0" &&
+	cat >expect <<-\EOF &&
+	Filter:
```

Comment:
I wonder what the point of the "Filter:" line is? Adds noise? We can have "No Filter" vs "Length: X\nData:..." be a clear enough split.